nixos/modules/services/mail/stalwart-mail.nix

```diff
@@ -48,6 +48,14 @@ in {
       ];
     };
 
+    users = {
```

Comment:
Could we add a comment explaining why we are adding a user here. It doesnt need to be too long, hiw about something like
"This service stores a potentially large amount of data. Running it as a dynamic user would force chown to be run everytime the service is restarted on a potentially large number of files. That would cause unecessary and unwanted delays"